app/controllers/reservations_controller.rb

```diff
@@ -138,6 +138,17 @@ def show
     @reservation  = Reservation.find(params[:id])
 
     raise ActiveRecord::RecordNotFound if @reservation != @order_detail.reservation
+
+    respond_to do |format|
+      format.html
+
+      format.ics do
+        calendar = ReservationCalendar.new(@reservation, request.host)
```

Comment:
I wouldn't mind verifying for HTTPS before we merge since we'll be dealing with it as soon as we put it on NU stage. Maybe use ngrok to test that out?